tests/components/filesize/test_sensor.py

```diff
@@ -69,3 +69,5 @@ async def test_state_unavailable(
 
     state = hass.states.get("sensor.file_txt_size")
     assert state.state == STATE_UNAVAILABLE
+
+    await mock_config_entry.async_unload(hass)
```

Comment:
Note: I was surprised to discover that config entries are not automatically unloaded on HA stop routine.
Is this an oversight, or is this on purpose?

Reply:
This is on purpose. Home Assistant either stops or a config entry unloads